cmd/thermal-recorder/config.go

```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"io/ioutil"
 
+	"github.com/TheCacophonyProject/thermal-recorder/location"
```

Comment:
The usual convention in Go is to group imports into 3 sections:
* standard library
* third party 
* local